Bottom: af32107058da460cd8e1965b187dd1c38b961556
Top:    9c6276fb1f4a9f78faca3cb229042bc3d99c77b9
Author: Ian Kent <ikent@redhat.com>
Date:   2013-04-03 16:16:28 +0800

Refresh of debug.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 1a71cb4..45ca99b 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1344,16 +1344,18 @@ static void *statemachine(void *arg)
 			master_mutex_lock();
 			if (list_empty(&master_list->completed)) {
 				if (list_empty(&master_list->mounts)) {
-					master_mutex_unlock();
 					debug(LOGOPT_ANY, "list empty completed and mounts");
+					master_mutex_unlock();
 					return NULL;
 				}
+				debug(LOGOPT_ANY, "list empty completed, do nothing");
 			} else {
 				if (master_done(master_list)) {
 					debug(LOGOPT_ANY, "master_done returns true");
 					master_mutex_unlock();
 					return NULL;
 				}
+				debug(LOGOPT_ANY, "master_done returns false");
 				master_mutex_unlock();
 				break;
 			}
